src/Injector.ts

```diff
@@ -131,6 +132,10 @@ export function Injector<C extends Evented, T extends Constructor<BaseInjector<C
 
 			return render();
 		}
+
+		public get registries(): RegistryHandler {
```

Comment:
Don't like this, this probably needs to be fixed up once the new registry changes are in.